relbench/data/task_base.py

```diff
@@ -118,7 +122,12 @@ def test_table(self) -> Table:
 
             full_table = self.make_table(
                 self.dataset._full_db,
-                pd.Series([self.dataset.test_timestamp]),
+                pd.date_range(
+                    self.dataset.test_timestamp,
+                    self.dataset.test_timestamp
+                    + self.timedelta * (self.dataset.max_eval_time_frames - 1),
+                    freq=self.timedelta,
+                ),
```

Comment:
Why don't we do the same for `val_tables`? It can also be enlarged in case the number of time frames is small, right?